Resources/config/services.yml

```diff
@@ -1,11 +1,28 @@
 services:
   werkspot.bing_ads_api_client:
     class: Werkspot\BingAdsApiBundle\Api\Client
-    arguments: ["@werkspot.request_new_access_token"]
+    arguments:
+      - "@werkspot.request_new_access_token"
+      - "@bing.client.proxy"
+      - "@bing.client.helper.zip"
+      - "@bing.client.helper.csv"
+
+  bing.client.proxy:
+    class: BingAds\Proxy\ClientProxy
+    arguments: ["example.com"]
+
+  bing.client.helper.zip:
+    class: Werkspot\BingAdsApiBundle\Api\Helper\Zip
+    arguments:
+      - "@guzzle.client"
+
+  bing.client.helper.csv:
+    class: Werkspot\BingAdsApiBundle\Api\Helper\Csv
 
   guzzle.client:
     class: GuzzleHttp\Client
 
   werkspot.request_new_access_token:
```

Comment:
the naming of the services is very inconsistent:

`Werkspot\BingAdsApiBundle\Api\Client => werkspot.bing_ads_api_client`
`Werkspot\BingAdsApiBundle\Api\Helper\Csv =>   bing.client.helper.csv:`
`Werkspot\BingAdsApiBundle\Guzzle\RequestNewAccessToken => werkspot.request_new_access_token`